components/validateur-report/numero-row.js

```diff
@@ -0,0 +1,63 @@
+import {useState, useMemo} from 'react'
+import PropTypes from 'prop-types'
+import {Pane, Text, UnorderedList, ListItem} from 'evergreen-ui'
+import {filter} from 'lodash'
+import {getLabel} from '@ban-team/validateur-bal'
+
+import AlertHeader from '@/components/validateur-report/alerts-header'
+import Dropdown from '@/components/dropdown'
+
+function NumeroRow({numero}) {
```

Comment:
L'objet `numero` n'est jamais utilisé tel quel, autant directement le déstructurer.

```suggestion
function NumeroRow({address, alerts}) {
```